operator/src/main/resources/application.properties

```diff
@@ -40,15 +40,12 @@ ingresscontroller.max-ingress-connections=108000
 # percentage of peak connections you actually need to meet
 ingresscontroller.peak-connection-percentage=100
 
-# Enables haproxy option contstats in the kas ingress so that stats are reported live rather than at connection close (RFE-3007 provide a proper mechanism to enable this option).
-ingresscontroller.ingress-container-command=/usr/bin/bash,-c,awk '{print $0} /^defaults$/ {print \"  option contstats\"}' < $TEMPLATE_FILE > /tmp/haproxy-config.template; exec /usr/bin/openshift-router --v=${ingresscontroller.router-verbosity} --template /tmp/haproxy-config.template --commit-interval=${ingresscontroller.commit-interval} --blueprint-route-pool-size=${ingresscontroller.blueprint-route-pool-size} --max-dynamic-servers=${ingresscontroller.max-dynamic-servers} --blueprint-route-namespace=${ingresscontroller.blueprint-namespace}
-# Disconnect established connections after a haproxy reconfiguration (NE-879 should eliminate the need for this)
-ingresscontroller.hard-stop-after=5s
-# Coalesce up-to reloadInterval worth of haproxy updates.  This prevents a flurry of haproxy restarts (1 per OpenShift Route) as each kafka
-# instance is created (NE-879 should eliminate the need for this).
-ingresscontroller.reload-interval-seconds=60
 ingresscontroller.router-verbosity=2
-ingresscontroller.blueprint-namespace=kas-route-blueprints
+# note that this variable is overridden using the downward API in the deployment.
+ingresscontroller.blueprint-namespace=kas-fleetshard-operator
```

Comment:
Since we use different namespace in stage is this still ok?

Reply:
the namespace will always be injected as an environment variable.  See the use of the downward API in the kubernetes.yaml